constants/itemsMap.js

```diff
@@ -1,15 +1,17 @@
-let items = require("./items.json")
-
 let itemsMap = new Map();
 
-for (let item of items) {
-    itemsMap.set(item.id, item);
+function setItems(items) {
+    itemsMap = new Map();
+    for (let item of items) {
+        itemsMap.set(item.id, item);

```

Comment:
Maybe we can filter out some of the item data we don't use to conserve memory? Otherwise, maybe we should add a way to input the items dictionary in the getNetworth method or something idk.

Reply:
I can also add that option is probably good to do